### PR TITLE
fix: Update Builder to Support goMod

### PR DESCRIPTION
**Pull request by kamirendawkins**

### CODE REVIEW

**Review by mwgrimm — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -1,17 +1,18 @@
 FROM golang:1.19-alpine3.15 as builder
```
Comment (mwgrimm) [inline Dockerfile]: could bump to 3.16 to match runner

File: Dockerfile [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
+
 WORKDIR ${GOPATH}/src/github.com/maxwellhealth/action-slack-reporter
-COPY main.go ${GOPATH}/src/github.com/maxwellhealth/action-slack-reporter
-ENV GOOS linux
-RUN go get -v ./...
-RUN go build   -o /go/bin/slack-reporter .
+
+COPY . .
+
+RUN go build   -o /go/bin/slack-reporter main.go
```
Comment (mwgrimm) [inline Dockerfile]: extra whitespace before `-o`

File: Dockerfile (hunk 1 above)
Comment (kamirendawkins) [reply Dockerfile]: Thanks, not sure how I didn't catch that before.

File: Dockerfile (hunk 2 above)
Comment (kamirendawkins) [reply Dockerfile]: Good catch, thanks!

**Review by mwgrimm — APPROVED** [review]

**Pull request merged**